src/document.js

```diff
@@ -177,8 +177,7 @@ class GeoPoint {
  * A DocumentSnapshot is an immutable representation for a document in a
  * Firestore database. The data can be extracted with
  * [data()]{@link DocumentSnapshot#data} or
- * [get(fieldPath)]{@link DocumentSnapshot#get} to get a
- * specific field.
+ * [get(fieldPath)]{@link DocumentSnapshot#get} (to get a specific field).
```

Comment:
As discussed offline, consider a separate sentence or other structure instead of adding `()`s, here and throughout.

Reply:
I've been trying to come up with a 2-sentence based approach that doesn't seem to clunky. I went back to what we had before for now, and just fixed the one actual issue in these docs.